aten/src/ATen/native/vulkan/ops/VulkanPackedContext.h

```diff
@@ -15,12 +15,14 @@ class VulkanPackedContext {
 
  public:
   VulkanPackedContext() : packed_{c10::AnyType::get()} {}
+  VulkanPackedContext(const VulkanPackedContext&) = default;
+  VulkanPackedContext(VulkanPackedContext&&) = default;
 
   inline const c10::IValue get_val(int64_t i) const {
     return packed_.get(i);
   }
 
-  inline void set_val(int64_t i, c10::IValue val) const {
+  inline void set_val(int64_t i, const c10::IValue& val) const {
     return packed_.set(i, val);
```

Comment:
Does generic list not have perfect forwarding here? If so, val should be std::move into set instead of being turned into a const ref (or alternatively perfectly forwarded).

Reply:
packed_.set() accepts const c10::IValue&.